### PR TITLE
Component testing

### PR DESCRIPTION
**Pull request by alex-latham**
Adds testing to the Ecotip and Configure components. The Ecotip (100% func coverage) does not successfully mock a response yet, so I'm definitely open to suggestions on how to make that happen. The Configure component (78.57% func cov) has sad path testing for out of range and missing values. Combined with the previous two PRs, this brings overall coverage up to 61% of functions.

## Type of change
Please uncheck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Add Testing
- [ ] Refactor

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/containers/Configure/Configure.js [hunk 1]
```diff
@@ -51,74 +51,80 @@ const Configure = ({ userQuote, allUserQuotes, loadSolarData, quote }) => {
...
               <div className="configure-form-item">
-                <label>System Size (kW): (1-10)</label>
-                <input
-                  type="number"
-                  placeholder="4"
-                  value={systemSize}
-                  required
-                  onChange={e => updateSystemSize(parseFloat(e.target.value))}
-                />
+                <label>System Size (kW) (1-10):
```
Comment (alex-latham) [inline src/containers/Configure/Configure.js:54]: I moved the : to the end of the label here

Comment (J-Poulter) [inline src/containers/Configure/Configure.js:54]: I like the way these input fields are nested in there labels.  I tried to implement that in the historical component but had to revert it due to complications with the way the styling was set up on that one

File: src/containers/Configure/Configure.test.js [hunk 2]
```diff
@@ -1,3 +1,202 @@
...
+    fireEvent.change(tiltInput, { target: { value: 100 } })
+    fireEvent.change(azimuthInput, { target: { value: 180 } })
+    fireEvent.click(submitButton)
+
+    const errMessage = getByText('Please reverify inputs to be in acceptable range')
+
+    expect(errMessage).toBeInTheDocument()
+  })
+
+  it.skip('should give an error message upon form submission if azimuth field is out of range', () => {
```
Comment (J-Poulter) [inline src/containers/Configure/Configure.test.js:179]: Great job testing all the different sad paths for this component.  Our testing is shaping up to look very thorough!

File: src/containers/Configure/Configure.test.js [hunk 3]
```diff
@@ -1,3 +1,202 @@
...
+    fireEvent.change(arrayTypeInput, { target: { value: 0 } })
+    fireEvent.change(systemLossesInput, { target: { value: 14 } })
+    fireEvent.change(tiltInput, { target: { value: 20 } })
+    fireEvent.change(azimuthInput, { target: { value: 375 } })
+    fireEvent.click(submitButton)
+
+    const errMessage = getByText('Please reverify inputs to be in acceptable range')
+
+    expect(errMessage).toBeInTheDocument()
+  })
```
Comment (nickstaylor) [inline src/containers/Configure/Configure.test.js:201]: This testing file is well detailed with checks for every input and error message.  Well done Alex!

File: src/containers/Configure/Configure.js (hunk 1 above)
Comment (alex-latham) [reply src/containers/Configure/Configure.js:54]: I can see how that would affect styling

**Pull request merged**